docs/src/DataStructures/RequestUrl.md

```diff
@@ -0,0 +1,3 @@
+```json
+"https://testnet.frequencyaccess.com/siwa/start?signedRequest=eyJyZXF1ZXN0ZWRTaWduYXR1cmVzIjp7InB1YmxpY0tleSI6eyJlbmNvZGVkVmFsdWUiOiJmNmNMNHdxMUhVTngxMVRjdmRBQk5mOVVOWFhveUg0N21WVXdUNTl0elNGUlc4eURIIiwiZW5jb2RpbmciOiJiYXNlNTgiLCJmb3JtYXQiOiJzczU4IiwidHlwZSI6IlNyMjU1MTkifSwic2lnbmF0dXJlIjp7ImFsZ28iOiJTcjI1NTE5IiwiZW5jb2RpbmciOiJiYXNlMTYiLCJlbmNvZGVkVmFsdWUiOiIweDM0OGZlZjIwNTY2OTE0YTc2MzMzN2VlZTUzNjc0YzZlZjVkYmEwM2FhMjczYjQ3ZTI4MDhkNTkxZjA5ZjkxMGQ3MDJjMWIxOWYwNmIwNjAzOWRlMzVhODNkNzNhYTU2MDIxODU3YTRlY2M5MjBjNjY2YjA0MjcwOTRmMWQ1MzgxIn0sInBheWxvYWQiOnsiY2FsbGJhY2siOiJodHRwOi8vbG9jYWxob3N0OjMwMDAiLCJwZXJtaXNzaW9ucyI6WzUsNyw4LDksMTBdfX0sInJlcXVlc3RlZENyZWRlbnRpYWxzIjpbeyJ0eXBlIjoiVmVyaWZpZWRHcmFwaEtleUNyZWRlbnRpYWwiLCJoYXNoIjpbImJjaXFtZHZteGQ1NHp2ZTVraWZ5Y2dzZHRvYWhzNWVjZjRoYWwydHMzZWV4a2dvY3ljNW9jYTJ5Il19LHsiYW55T2YiOlt7InR5cGUiOiJWZXJpZmllZEVtYWlsQWRkcmVzc0NyZWRlbnRpYWwiLCJoYXNoIjpbImJjaXFlNHFvY3poZnRpY2k0ZHpmdmZiZWw3Zm80aDRzcjVncmNvM29vdnd5azZ5NHluZjQ0dHNpIl19LHsidHlwZSI6IlZlcmlmaWVkUGhvbmVOdW1iZXJDcmVkZW50aWFsIiwiaGFzaCI6WyJiY2lxanNwbmJ3cGMzd2p4NGZld2NlazVkYXlzZGpwYmY1eGppbXo1d251NXVqN2UzdnUydXducSJdfV19XX0&callbackUrlParams=id%3D11223344"
```

Comment:
for the callbackUrlParams is the intent for it to be like a HTML checkbox, meaning `callbackUrlParams=id%3D11223344&callbackUrlParams=foo%3Dbar` or is it just a query string fragment (I think this is how the JS example app is handling it), e.g. `callbackUrlParams=id%3D11223344%26foo%3Dbar` or a delimited list of some sort like (delimiter is a comma %2C here) `callbackUrlParams=id%3D11223344%2Cfoo%3Dbar`